client/python/examples/README.md

```diff
@@ -0,0 +1,265 @@
+# Python Client Examples
+
+Currently there are three example files:
+
+- [simple.py](#simple.py)
+- [general.py](#general.py)
+- [queues.py](#queuespy)
+- [cancelling.py](#cancellingpy)
+- [monitor.py](#monitor.py)
+
+There are also some codeblocks with useful code snippets:
```

Comment:
I’d make this reflect what you trying to show with auth ie this section could reflect examples with more complicated auth